src/modules/landing/Landing.jsx

```diff
@@ -10,87 +12,15 @@ export default class Landing extends React.Component {
 
 	render() {
 		return (
-			<LandingInner>
+			<div className="landing-inner">
 				<h1>Listen</h1>
-				<Circle id="c1" onClick={this.props.closeLanding} />
-				<Circle id="c2" />
-				<Circle id="c3" />
-				<Circle id="c4" />
-				<Circle id="c5" />
-				<Circle id="c6" />
-			</LandingInner>
+				<div id="c1" onClick={this.props.closeLanding} />
```

Comment:
Low Priority:
It's much better to use classnames instead of ids because ids have a very high [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

This makes it really hard to override the styling later.